compute_endpoint/tests/unit/test_working_dir.py

```diff
@@ -10,17 +10,30 @@
     ThreadPoolEngine,
 )
 from globus_compute_endpoint.engines.helper import execute_task
-from parsl.executors import HighThroughputExecutor
 from tests.utils import get_cwd
 
 
-@pytest.fixture()
+@pytest.fixture
 def reset_cwd():
     cwd = os.getcwd()
     yield
     os.chdir(cwd)
 
 
+@pytest.fixture
+def mock_gce():
```

Comment:
``mock_gce`` can be confusing, especially compared with the bare ``gce`` above.  'e' can be executor or engine, and you are mocking one of them here.  I feel like being more specific for gce and mock_gce (and making the name be the explanation more) would reduce ambiguity.

Reply:
Replaced with `mock_gcengine`